docs/10-WebServices.md

```diff
@@ -67,6 +107,54 @@ class ApiHelper extends \Codeception\Module
 
 The same way you can receive request parameters and headers.
 
+### Validate JSON structures
+
+It is pretty common for API tests to not only validate the received data but to check the structure of the response. Response data is not something constant, and may change on each request, however the JSON/XML structure should kept the same for an API version. In order to check response structure REST module have some useful methods. 
```

Comment:
Response data is not **~~something constant~~usually considered to be consistent**, and may change on each request,

structure should **be** kept the same for an API

check response structure **the** REST module **~~have~~has** some useful methods